src/containers/Tenant/Query/ExplainResult/components/SimplifiedPlan/MetricsCell.tsx

```diff
@@ -0,0 +1,19 @@
+import {isNumeric, toExponential} from '../../../../../../utils/utils';
+
+import {block} from './utils';
+
+interface MetricsCellProps {
+    value: unknown;
+}
+
+export function MetricsCell({value}: MetricsCellProps) {
+    if (!isNumeric(value)) {
+        return undefined;
+    }
+
+    const numberValue = Number(value);
+    const content =
+        numberValue < 1e8 ? Math.round(numberValue).toLocaleString() : toExponential(value, 3);
```

Comment:
If I expand schema tree too much, values start to overlap each other

<img width="878" alt="Screenshot 2024-07-25 at 22 09 09" src="https://github.com/user-attachments/assets/3a5a9bd4-3949-4ce1-9df1-2ca794248f45">